src/editor/render.jsx

```diff
@@ -10,6 +10,22 @@ const makeFootnote = (footnote) => {
   return free;
 };
 
+const toggleAccordionReference = (footnoteId) => {
+  if (
+    typeof window !== 'undefined' &&
+    document.querySelector(footnoteId) !== null &&
+    document.querySelector(footnoteId).closest('.accordion') !== null
```

Comment:
@danielamormocea This should be put within a `const` in order not to query it twice.